src/constants.js

```diff
@@ -4,3 +4,8 @@
  * This source code is licensed under the Apache 2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
+
```

Comment:
A more general question: All these constants are only used in one place. Why have they split away into a separate file? Just define them in the head of each file where it is been needed.